src/main/java/org/terasology/web/resources/games/AbstractAction.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.web.resources.games;
+
+import org.terasology.engine.module.ModuleManager;
+import org.terasology.engine.paths.PathManager;
+import org.terasology.web.resources.ResourceAccessException;
+
+public abstract class AbstractAction {
```

Comment:
I always try to make an abstract class implement an interface, to allow implementors to provide alternative content. Since this class just provides trivial code, it could be moved into the actual implementation and converted directly into an interface. But this would require removing the `setPathManager` method for the tests. Hmm .. tricky.